cargo-shuttle/src/args.rs

```diff
@@ -1,116 +1,122 @@
 use std::{
-    ffi::{OsStr, OsString},
+    ffi::OsStr,
     fs::{canonicalize, create_dir_all},
+    io::{self, ErrorKind},
     path::PathBuf,
 };
 
+use clap::Parser;
 use shuttle_common::project::ProjectName;
-use structopt::StructOpt;
 
-#[derive(StructOpt)]
-#[structopt(
+#[derive(Parser)]
+#[clap(
     // Cargo passes in the subcommand name to the invoked executable. Use a
     // hidden, optional positional argument to deal with it.
-    arg(structopt::clap::Arg::with_name("dummy")
+    arg(clap::Arg::with_name("dummy")
         .possible_value("shuttle")
         .required(false)
         .hidden(true))
 )]
 pub struct Args {
-    #[structopt(
+    #[clap(
         long,
-        about = "allows targeting a custom deloyed instance for this command only",
+        help = "allows targeting a custom deloyed instance for this command only",
```

Comment:
Maybe we should turn the `help` attribute into a documentation comment like they are in provisioner?

Reply:
Sure thing!

Reply:
Done!